manifests/f/FiloSottile/mkcert/1.4.4/FiloSottile.mkcert.installer.yml

```
@@ -0,0 +1,13 @@
+# yaml-language-server: $schema=https://github.com/microsoft/winget-cli/raw/master/schemas/JSON/manifests/v1.2.0/manifest.installer.1.2.0.json
+PackageIdentifier: FiloSottile.mkcert
+PackageVersion: 1.4.4
+Installers:
+  - Architecture: x64
+    InstallerUrl: https://github.com/FiloSottile/mkcert/releases/download/v1.4.4/mkcert-v1.4.4-windows-amd64.exe
+    InstallerSha256: D2660B50A9ED59EADA480750561C96ABC2ED4C9A38C6A24D93E30E0977631398
+  - Architecture: arm64
+    InstallerUrl: https://github.com/FiloSottile/mkcert/releases/download/v1.4.4/mkcert-v1.4.4-windows-arm64.exe
+    InstallerSha256: 793747256C562622D40127C8080DF26ADD2FB44C50906CE9DB63B42A5280582E
+InstallerType: portable
+ManifestType: installer
+ManifestVersion: "1.2.0"
```

```suggestion
ManifestVersion: 1.2.0

```
Newline preferred at end of file